packages/console/src/pages/CheckoutSuccessCallback/index.tsx

```diff
@@ -74,22 +80,45 @@ function CheckoutSuccessCallback() {
   );
 
   const isCheckoutSuccessful =
-    !isLoadingPlans &&
     checkoutTenantId &&
     stripeCheckoutSession.status === 'complete' &&
-    checkoutPlanId === tenantSubscription?.planId;
+    (isDevFeaturesEnabled
+      ? !isLoadingLogtoSkus && checkoutSkuId === tenantSubscription?.planId
+      : !isLoadingPlans && checkoutPlanId === tenantSubscription?.planId);
 
   useEffect(() => {
     if (isCheckoutSuccessful) {
       clearLocalCheckoutSession();
 
-      const checkoutPlan = subscriptionPlans?.find((plan) => plan.id === checkoutPlanId);
-      if (checkoutPlan) {
-        toast.success(
-          <Trans components={{ name: <PlanName name={checkoutPlan.name} /> }}>
-            {t(isDowngrade ? 'downgrade_success' : 'upgrade_success')}
-          </Trans>
-        );
+      if (isDevFeaturesEnabled) {
+        const checkoutSku = logtoSkus?.find((sku) => sku.id === checkoutPlanId);
+        if (checkoutSku) {
+          toast.success(
+            <Trans
+              components={{
+                name: (
+                  <PlanName
+                    skuId={checkoutSku.id}
+                    // Generally `checkoutPlanId` and a properly setup of SKU `name` should not be null, we still need to handle the edge case to make the type inference happy.
+                    // Also `name` will be deprecated in the future once the new pricing model is ready.
+                    name={checkoutPlanId ?? checkoutSku.name ?? checkoutSku.id}
```

Comment:
Shouldn't sku name come first?

Reply:
Should have planId before SKU name since the `<PlanName />` component will look up admin console key for given plan ID and then use the raw value (of input argument `name`) as fallback.